src/components/input/elastic-textarea.test.ts

```diff
@@ -75,3 +75,46 @@ test(
     await expect(textarea).toHaveAttribute('rows', '1');
   })
 );
+
+test(
+  'Should update to fit content when textarea is resized',
+  withBrowser(async ({ utils, screen, page, waitFor }) => {
+    await loadGlobalCSS(utils);
+    await utils.injectHTML(
+      await textInputHTML({
+        class: 'js-elastic-textarea',
+        type: 'textarea',
+        value:
+          'We are a small, versatile team who care passionately about the web. We’re full of what our industry considers unicorns. Our designers code. Our developers went to art school.',
+      })
+    );
+    const textarea = await screen.getByRole('textbox');
+    await initTextareaJS(utils, textarea);
+
+    // First test the viewport changing size, start with a wide viewport
+    await page.setViewport({ width: 800, height: 400 });
+    // The text should wrap two lines
+    await expect(textarea).toHaveAttribute('rows', '2');
+
+    // Then size it down to a narrow viewport width
+    await page.setViewport({ width: 250, height: 400 });
+    // The textarea should resize forcing the text to wrap multiple lines
```

Comment:
For some reason my brain kept trying to read "resize forcing" as one thing instead of as parts of separate clauses.

```suggestion
    // The textarea should resize, forcing the text to wrap multiple lines
```